x/protorev/keeper/protorev.go

```diff
@@ -146,6 +146,57 @@ func (k Keeper) DeleteAllPoolsForBaseDenom(ctx sdk.Context, baseDenom string) {
 	k.DeleteAllEntriesForKeyPrefix(ctx, key)
 }
 
+// SetSwapsToBackrun sets the swaps to backrun, updated via hooks
+func (k Keeper) SetSwapsToBackrun(ctx sdk.Context, swapsToBackrun types.Route) error {
+	store := prefix.NewStore(ctx.KVStore(k.storeKey), types.KeyPrefixSwapsToBackrun)
+
+	bz, err := swapsToBackrun.Marshal()
+	if err != nil {
+		return err
+	}
+
+	store.Set(types.KeyPrefixSwapsToBackrun, bz)
```

Comment:
nit: we also have`osmoutils.Set` and `osmoutils.MustSet`. Feel free to resolve, non-blocking

Reply:
Noting this down for a future PR to switch all the setters/getters in protorev using these. Resolving